### PR TITLE
feat(org-stats-9): Improve date display and add some copy

### PR DESCRIPTION
**Pull request by leedongwei**
image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/usageStats/usageStatsProjects.tsx [hunk 1]
```diff
@@ -169,14 +169,16 @@ class UsageStatsProjects extends AsyncComponent<Props, State> {
...
 
     if (dataCategory === DataCategory.TRANSACTIONS) {
-      return `/organizations/${organization.slug}/performance/`;
+      return `/organizations/${organization.slug}/performance/?project=${project.id}`;
     }
 
-    return `/organizations/${organization.slug}/issues/`;
+    return organization.features.includes('project-detail')
+      ? `/organizations/${organization.slug}/projects/${project.slug}/?project=${project.id}`
+      : `/organizations/${organization.slug}/issues/?project=${project.id}`;
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/usageStats/usageStatsProjects.tsx:181]: Do you need to preserve the current time range?

Comment (leedongwei) [reply src/sentry/static/sentry/app/views/usageStats/usageStatsProjects.tsx:181]: Fixed in https://github.com/getsentry/sentry/pull/25211/commits/4c2171fe2f0f5fd8f9785b7cf300db966d3f63ff

File: src/sentry/static/sentry/app/views/usageStats/usageChart/utils.tsx [hunk 2]
```diff
@@ -6,11 +6,27 @@ import {intervalToMilliseconds} from 'app/utils/dates';
...
+export const FORMAT_DATETIME_HOURLY = 'MMM D LT';
+export const FORMAT_DATETIME_DAILY = 'MMM D';
 
-  return unit > 1 ? m.startOf('h').format('MMM D') : m.startOf('h').format('MMM D HH:mm');
+/**
+ * Used to generate X-axis data points and labels for UsageChart
+ * Ensure that this method is idempotent and doesn't change the moment object
+ * that is passed in
+ */
+export function getDateFromMoment(m: moment.Moment, interval: IntervalPeriod = '1d') {
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/usageStats/usageChart/utils.tsx:21]: Should we consider having a similar date formatter for other charts?

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/organizations/headerItem.tsx [hunk 3]
```diff
@@ -189,7 +189,7 @@ const StyledChevron = styled(IconChevron, {shouldForwardProp: isPropValid})<{
   color: ${getColor};
 `;
 
-const SettingsIconLink = styled(Link)`
+export const SettingsIconLink = styled(Link)`
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/organizations/headerItem.tsx:192]: We should move this into a new file if it's being imported outside of `HeaderItem`

File: src/sentry/static/sentry/app/views/usageStats/usageChart/utils.tsx [hunk 4]
```diff
@@ -1,26 +1,56 @@
...
+export const FORMAT_DATETIME_HOURLY = 'MMM D LT';
+export const FORMAT_DATETIME_DAILY = 'MMM D';
+
+/**
+ * Used to generate X-axis data points and labels for UsageChart
+ * Ensure that this method is idempotent and doesn't change the moment object
+ * that is passed in
+ */
+export function getDateFromMoment(m: moment.Moment, interval: IntervalPeriod = '1d') {
+  const days = intervalToMilliseconds(interval) / (1000 * 60 * 60 * 24);
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/usageStats/usageChart/utils.tsx:22]: Should we make `(1000 * 60 * 60 * 24)` into a module level constant?

Comment (leedongwei) [reply src/sentry/static/sentry/app/views/usageStats/usageChart/utils.tsx:22]: This is fixed in https://github.com/getsentry/sentry/pull/25211/ (https://github.com/getsentry/sentry/pull/25211/commits/44c72a6fd04a2a07645ceccc83d34a9b5fee8631)

File: src/sentry/static/sentry/app/components/organizations/headerItem.tsx (hunk 3 above)
Comment (leedongwei) [reply src/sentry/static/sentry/app/components/organizations/headerItem.tsx]: Good point. I'll try to find a home for it.

**Pull request merged**